helm/idol-library/templates/aciserver/_statefulset.tpl

```diff
@@ -18,6 +18,8 @@
 {{ $component := get . "component" | required "idol-library.aciserver.statefulset.base.v1: missing component" }}
 {{ $volumes := get . "volumes" | default list }}
 {{ $containers := get . "containers" | default (list "idol-library.aciserver.container.base.v1") }}
+{{ $initContainers := get . "initContainers" | default list }}
```

Comment:
I don't think we need `initContainers` as a parameter here - since we don't define any default initContainers (and seem unlikely to?) these can just be supplied via the statefulset merge